File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/discoverHere/LockupEntrypoint/OpeningHours.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class OpeningHours {

    @SerializedName("text")
    @Expose
    private String text;
    @SerializedName("label")
    @Expose
    private String label;
    @SerializedName("isOpen")
    @Expose
    private Boolean isOpen;
    @SerializedName("structured")
    @Expose
    private List<Structured> structured = null;

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getLabel() {
        return label;
    }

    public void setLabel(String label) {
        this.label = label;
    }

    public Boolean getIsOpen() {
        return isOpen;
    }

    public void setIsOpen(Boolean isOpen) {
        this.isOpen = isOpen;
    }

    public List<Structured> getStructured() {
        return structured;
    }

    public void setStructured(List<Structured> structured) {
        this.structured = structured;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverHere/LockupEntrypoint/OpeningHours.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint

import com.google.gson.annotations.Expose

class OpeningHours {
    @SerializedName("text")
    @Expose
    private var text: String? = null

    @SerializedName("label")
    @Expose
    private var label: String? = null

    @SerializedName("isOpen")
    @Expose
    private var isOpen: Boolean? = null

    @SerializedName("structured")
    @Expose
    private var structured: List<com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Structured>? = null
    fun getText(): String? {
        return text
    }

    fun setText(text: String?) {
        this.text = text
    }

    fun getLabel(): String? {
        return label
    }

    fun setLabel(label: String?) {
        this.label = label
    }

    fun getIsOpen(): Boolean? {
        return isOpen
    }

    fun setIsOpen(isOpen: Boolean?) {
        this.isOpen = isOpen
    }

    fun getStructured(): List<com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Structured>? {
        return structured
    }

    fun setStructured(structured: List<com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Structured>?) {
        this.structured = structured
    }
}